scripts/install-helpers/baremetal-coco/nfd/intel-rules.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: nfd.openshift.io/v1alpha1
+kind: NodeFeatureRule
+metadata:
+  name: intel-dp-devices
+  namespace: openshift-nfd
+spec:
+  rules:
+    - name: "intel.gpu"
+      labels:
+        "intel.feature.node.kubernetes.io/gpu": "true"
```

Comment:
This is supposed to be about CoCo, right ? Can you clarify  why we care for a GPU here ?

Reply:
Good point. I think this comes from confidential GPU. But right now it doesn't work with OSC anyway .. So I'll remove it